files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py

```diff
@@ -27,7 +27,7 @@
 import SettingsWidgets
 
 # i18n
-gettext.install("cinnamon", "/usr/share/locale", names="ngettext")
+gettext.install("cinnamon", "/usr/share/locale", names="gettext.ngettext")
```

Comment:
Both `"ngettext"` and `"gettext.ngettext"` are incorrect, they are the set sequence `{'t', 'x', 'e', 'n', 'g', '.'}` and have no syntactic meaning here. Additionally, your later on ngettext is changed to be accessed via gettext.ngettext, so it doesn't need the install() function to install a global namespace object.